src/citation/serializers/citation_project_serializer.py

```diff
@@ -27,11 +27,21 @@ def get_current_user_is_admin(self, project_instance):
         current_user = self.context.get("request").user
         return project_instance.get_is_user_admin(current_user)
 
-    def get_editors(self, project):
-        editor_ids = project.permissions.filter(access_type=EDITOR).values_list("user")
-        return MinimalUserSerializer(
-            User.objects.filter(id__in=editor_ids), many=True
-        ).data
+    def get_collaborators(self, project):
+        editor_ids = project.permissions.filter(access_type__in=[EDITOR]).values_list(
```

Comment:
Is there a reason for filtering by a list of access type instead of just access type itself? Same applies for the filtering below